shared-modules/SDL/SDL-1.2.15.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This should be a git submodule.

Reply:
I'm not familiar with submodules, does it means I should only push the .gitmodules file and not the shared_modules folder?